site/js/modules/webui/server.js

```diff
@@ -0,0 +1,190 @@
+import * as Autograder from '../autograder/base.js'
+
+import * as Render from './render.js'
+import * as Routing from './routing.js'
+
+/* The priority of the field to show first. */
+/* Items later in the list have the highest priority. */
```

Comment:
Favor `//` for comments.
(This makes it easier to use block comments to block out large chunks of code.)